gallery/gallery/lib/studies/crane/item_cards.dart

```diff
@@ -33,8 +34,13 @@ class _ItemCardsState extends State<ItemCards> {
 
     return destinations
         .map(
-          (d) => RepaintBoundary(
-            child: _DestinationCard(destination: d),
+          (d) => HighlightFocus(
+            debugLabel: 'DestinationCard: ${d.destination}',
+            highlightColor: Colors.red.withOpacity(0.5),
+            onPressed: () => print('pressed'),
```

Comment:
Should the pressed here be removed? Or why do we have it?

Reply:
Yes it should.